x/authz/module/autocli.go

```diff
@@ -47,7 +48,18 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			},
 		},
 		Tx: &autocliv1.ServiceCommandDescriptor{
-			Service: authzv1beta1.Msg_ServiceDesc.ServiceName,
+			Service:              authzv1beta1.Msg_ServiceDesc.ServiceName,
+			EnhanceCustomCommand: true,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod:      "Revoke",
+					Use:            "revoke [grantee] [msg-type-url] --from=[granter]",
+					Short:          `Revoke authorization from a granter to a grantee`,
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "grantee"}, {ProtoField: "msg_type_url"}},
```

Comment:
very very tiny nit, can we put `Example` above `PositionalArgs` for consistency with the rest.